site/content/en/docs/Guides/Client SDKs/rust.md

```diff
@@ -52,6 +52,13 @@ To mark the [game session as ready]({{< relref "_index.md#ready" >}}) call `sdk.
 sdk.ready()?;
 ```
 
+
+{{% feature publishVersion-="1.0.0" %}}To mark that the [reserve the game session]({{< relref "_index.md#reserve-seconds" >}}) for a duration call `sdk.reserve(duration)`.
```

Comment:
This doesn't read very well:

> To mark that the reserve the game session for a duration call sdk.reserve(duration).

Maybe something like "To mark the game server as reserved for a period of time, call reserve(duration)." (this also matches the nodejs documentation).

Reply:
I don't even know what I was typing on this one. Lemme fix this now, better to have it for 1.0

Reply:
Done!